backend/controllers/settings.py

```diff
@@ -55,8 +66,33 @@ async def get_settings(keys: str):
 
 @router.post("/set")
 async def set_settings(settings: List[SetSettings]):
-    settings_manager.set(settings)
+    current_settings = settings_manager.settings.dict()

```

Comment:
Using path strings is something I moved away from in the backend since we can just refer to settings with actual object calls now.  To keep backwards compatibility I added in similar logic for the API.